Update color palette for accessibility

kartikcho opened the pull request:
The current color palette is too similar for people with color deficiency people to differentiate. Made hover colors much darker in contrast and set user timing marks to grey color.

This is a re-PR that addresses comments from #113 and resolves #55 

![image](https://user-images.githubusercontent.com/48270786/89783584-d06fed80-db34-11ea-985c-8a457bb086e4.png)

kartikcho committed:
Update color palette

taneliang commented:
The new user timing mark colors are nice!

I like that the yellows have been reverted. The colors are also almost the same as the ones on master, which I guess is a good thing? I didn't have a problem with the blue hover colors on master, so I can't really say if this fixed any accessibility issues (I didn't run this through any color filters so I'm not sure if people with protanopia or other color deficiencies will appreciate any of the changes here).

Unfortunately, this does mean that the accessibility issues I had from master aren't fixed in this PR:

* The hovered/unhovered reds are too similar, similar enough that they don't look like they changed color when I hover over them
* (Comment from the last PR) The unhovered render blue and the layout effect blue are an identical sky blue

Happy to approve this as is even though it doesn't resolve #55 imo. We can always make another PR in the future to resolve that.

kartikcho committed:
Increase color palette contrast

kartikcho commented:
I've updated the colors again. Red is significantly darker now on hover. Also layout effects are purple but almost similar in contrast with unhovered render blues. 

I've changed the contrast of layout effects but if you're still having trouble, you can select a shade for it. 

Let me know if you can distinguish now @taneliang!

taneliang commented on src/content-views/constants.js:
```diff
@@ -48,17 +48,17 @@ export const COLORS = Object.freeze({
   REACT_LAYOUT_EFFECTS_SELECTED: '#934FC1',
   REACT_LAYOUT_EFFECTS_HOVER: '#601593',
-  REACT_PASSIVE_EFFECTS: '#c49edd',
+  REACT_PASSIVE_EFFECTS: '#d8aff4',
```
Was it intentional for passive and layout effect colors to only differ when they're not hovered over?

kartikcho replied:
Ah, good catch. must've slipped through. No, they're supposed to be same.

taneliang reviewed (approved):
Nice! I like the new reds, they're so much better.

kartikcho committed:
Set passive effects color same as layout effects

kartikcho committed:
Merge branch 'master' into color-a11y
